setup.cfg

```diff
@@ -27,20 +27,19 @@ zip_safe = False
 packages = find:
 include_package_data = True
 install_requires =
-    dvc
+    dvc@git+https://github.com/iterative/dvc
     gcsfs>=2022.7.1
 
 [options.extras_require]
 tests =
     wheel==0.37.0
-    dvc
     # Test requirements
-    pytest==6.2.5
+    pytest>=7.1.2
```

Comment:
Do we actually depend on some newer features?

Reply:
This is just due to `pytest-servers` requiring `pytest==7.1.2`, although I think we could lower the requirement.

Reply:
Gotcha. I guess pytest-servers also has an unnecessarily strict requirement for pytest, right? Maybe we could relax it in both places. But the current change will do fine for now too.